app/views/govuk_publishing_components/components/docs/cookie_banner.yml

```diff
@@ -16,7 +18,7 @@ examples:
   with_custom_content:
     data:
       title: "Can we store analytics cookies on your device?"
-      text: "<p>This is some custom text in my cookie banner which lets users know what we're using cookies for. I can also include a link to the <a class='govuk-link' href='/cookies'>cookies page</a></p>"
+      text: "This is some custom text in my cookie banner which lets users know what we're using cookies for. I can also include a link to the <a class='govuk-link' href='/cookies'>cookies page</a>"
```

Comment:
Probably worth adding another example like this one to show passing multiple strings for the text as an array, and including a test for that (if there isn't one already).

Reply:
added